charts/seed-bootstrap/charts/loki/values.yaml

```diff
@@ -12,7 +12,7 @@ authEnabled: true
 annotations: {}
 
 labels:
-  # deprecated label only for the health check
+  # deprecated label
   garden.sapcloud.io/role: logging
```

Comment:
Why is this not yet removed?

Reply:
Well, the `labels` are also used in the loki's Statefulset `spec.selector` which is immutable field. To workaround a similar issue with immutability of `spec.selector` for Deployment in https://github.com/gardener/gardener/pull/3264 I introduced a helper func that deletes the Deployment to allow new Deployment to be create right away with the desired spec. The same approach I think is not applicable for StatefulSets as they bring a PVC which we don't want to delete.

Reply:
When a `StatefulSet` is deleted then the PVC remains in the system (at least not with our configuration), so I guess we can follow the same approach. We can do that separately, of course, so I'm going to approve the PR for now. Thanks!

Reply:
Okay, I will give a try. Thank you!

Reply:
I now updated the PR. Thank you once again!